h/views/api/flags.py

```diff
@@ -18,19 +19,23 @@
 def create(context, request):
     request.find_service(name="flag").create(request.user, context.annotation)
 
-    _email_group_admin(request, context.annotation)
+    _email_group_moderators(request, context.annotation)
 
     return HTTPNoContent()
 
 
-def _email_group_admin(request, annotation):
+def _email_group_moderators(request, annotation):
     incontext_link = links.incontext_link(request, annotation)
     if incontext_link is None:
         incontext_link = annotation.target_uri
 
-    group = annotation.group
-    if group.creator and group.creator.email:
-        send_params = flag_notification.generate(
-            request, group.creator.email, incontext_link
-        )
-        mailer.send.delay(*send_params)
+    group_members_service = request.find_service(name="group_members")
+
+    memberships = group_members_service.get(
+        annotation.group, roles=list(GROUP_MODERATE_PREDICATES.keys())
```

Comment:
Not sure how often/ how flexible this will be in the future but maybe listing the `[
GroupMembershipRoles.OWNER,  GroupMembershipRoles.ADMIN, GroupMembershipRoles.MODERATOR]`

here will just read better.